api/src/pcapi/routes/backoffice_v3/templates/components/search/offer/forms.html

```diff
@@ -0,0 +1,43 @@
+{% macro build_offer_modofication_form(offer_id, offer_name, url, csrf_token, ponderation, tags) %}
+    <form action="{{ url }}" method="post" name="edit-offer-modal-form-{{ offer_id }}" class="modal-content">
+        {{ csrf_token }}
+        <div class="modal-header">
+            <h5 class="modal-title">Modification de l'offre <span
+                    class="text-secondary">{{ offer_name }}</span></h5>
+            <button class="btn-close" type="button" data-bs-dismiss="modal"
+                    aria-label="Fermer"></button>
+        </div>
+        <div class="modal-body">
+            <div class="flex">
+                <div class="form-floating criteria-{{ offer_id }}-container">
+                    <select
+                            multiple
+                            data-tomselect-autocomplete-url="{{ url_for('backoffice_v3_web.autocomplete_criteria') }}"
```

Comment:
```suggestion
                            data-tomselect-autocomplete-url="{{ field.tomselect_autocomplete_url }}"
```

Voir : https://github.com/pass-culture/pass-culture-main/blob/master/api/src/pcapi/routes/backoffice_v3/forms/fields.py#L154-L164